site/_docs/templates.md

```diff
@@ -286,10 +286,10 @@ root of your source directory. This will embed the contents of
   </p>
 </div>
 
-You can also pass parameters to an include:
+You can also pass parameters to an include. Note that the variable parameter is NOT surrounded with the usual liquid curly brackets:
 
 {% highlight ruby %}
-{% raw %}{% include footer.html param="value" %}{% endraw %}
+{% raw %}{% include footer.html param="value" variable-param=page.variable %}{% endraw %}
```

Comment:
It is totally possible that I'm wrong, but I *think* dashes are not allowed in Liquid variables...